src/managed_data/client.py

```diff
@@ -72,12 +70,33 @@ def __str__(self):
 class DeepOriginClient(Client):
     """client to interact with DeepOrigin API"""
 
-    api_url = _nucleus_url()
-    org_id = get_value()["organization_id"]
-
     headers = dict()
 
-    def authenticate(self, refresh_tokens: bool = True):
+    def __init__(
+        self,
+        *,
+        api_url: Optional[str] = None,
+        org_id: Optional[str] = None,
+    ):
+        """custom init method to create a DeepOriginClient without needing config files"""
+        if not api_url:
+            from deeporigin.utils import _nucleus_url
+
+            api_url = _nucleus_url()
+
+        if not org_id:
+            from deeporigin.config import get_value
+
+            org_id = get_value()["organization_id"]
+
+        super().__init__()
+
+    def authenticate(
+        self,
+        *,
+        refresh_tokens: bool = True,
+        access_token: Optional[str] = None,
+    ):
         """authenticate to DeepOrigin API
```

Comment:
the authenticate method now accepts an access token, and if provided, does nothing